src/lib/EditorToolbar.js

```diff
@@ -61,22 +58,22 @@ export default class EditorToolbar extends Component {
   }
 
   render() {
-    const {className} = this.props;
+    const {className, toolbarConfig} = this.props;
     return (
       <div className={cx(styles.root, className)}>
-        {this._renderInlineStyleButtons()}
-        {this._renderBlockTypeButtons()}
+        {this._renderInlineStyleButtons(toolbarConfig || EditorToolbarConfig)}
+        {this._renderBlockTypeButtons(toolbarConfig || EditorToolbarConfig)}
         {this._renderLinkButtons()}
-        {this._renderBlockTypeDropdown()}
+        {this._renderBlockTypeDropdown(toolbarConfig || EditorToolbarConfig)}
```

Comment:
I realize now we should call this `DefaultEditorToolbarConfig`, but that's not too important.